### PR TITLE
[#962] Add stub variables

### PR DESCRIPTION
**Pull request by jhflorey**
This PR addresses issue https://github.com/episphere/connect/issues/962

### CODE REVIEW

File: utils/fieldToConceptIdMapping.js [hunk 1]
```diff
@@ -300,6 +300,30 @@ module.exports = {
...
+        unreliableInternetDevice: 121430614,
+        worriedAboutStudyResults: 523768810,
+        worriedStudyWillFindSomethingBad: 639172801,
+        privacyConcerns: 175732191,
+        donNotTrustGov: 150818546,  
+        donNotTrustRsrchrs: 624030581,  
+        donNotWantInfoSharedWithRsrchrs: 285488731,  
+        worriedInfoNotSecure: 596510649,
+        worriedInsWillGetData: 866089092,
+        iAmWorriedAboutDataBeingGivenToMyEmployerPotentialEmployer: 990579614,
```
Comment (we-ai) [inline utils/fieldToConceptIdMapping.js:315]:
```suggestion
        worriedAboutDataBeingGivenToMyEmployerPotentialEmployer: 990579614,
```

File: utils/fieldToConceptIdMapping.js [hunk 2]
```diff
@@ -300,6 +300,30 @@ module.exports = {
...
+        worriedAboutDiscriminationFromData: 131458944,
+        worriedOthersWillProfitFromMyData: 372303208,
+        otherPrivacyConcerns: 777719027,
+        unAbleToCompleteOnlineActivities: 620696506,
```
Comment (we-ai) [inline utils/fieldToConceptIdMapping.js:319]:
```suggestion
        unableToCompleteOnlineActivities: 620696506,
```

File: utils/fieldToConceptIdMapping.js [hunk 3]
```diff
@@ -300,6 +300,30 @@ module.exports = {
...
+        donNotLikeThingsOnline: 352891568,
```
Comment (we-ai) [inline utils/fieldToConceptIdMapping.js:320]:
```suggestion
        doNotLikeThingsOnline: 352891568,  
```

File: utils/fieldToConceptIdMapping.js [hunk 4]
```diff
@@ -300,6 +300,30 @@ module.exports = {
...
+        donNotWantInfoSharedWithRsrchrs: 285488731,  
+        worriedInfoNotSecure: 596510649,
+        worriedInsWillGetData: 866089092,
+        iAmWorriedAboutDataBeingGivenToMyEmployerPotentialEmployer: 990579614,
+        worriedAboutDiscriminationFromData: 131458944,
+        worriedOthersWillProfitFromMyData: 372303208,
+        otherPrivacyConcerns: 777719027,
+        unAbleToCompleteOnlineActivities: 620696506,
+        donNotLikeThingsOnline: 352891568,  
+        covConcerns: 958588520,
```
Comment (we-ai) [inline utils/fieldToConceptIdMapping.js:321]:
```suggestion
        covidConcerns: 958588520,
```

File: utils/fieldToConceptIdMapping.js [hunk 5]
```diff
@@ -300,6 +300,30 @@ module.exports = {
...
+        partNowUnAbleToParticipate: 875010152,
```
Comment (we-ai) [inline utils/fieldToConceptIdMapping.js:322]:
```suggestion
        partNowUnableToParticipate: 875010152,
```

File: utils/fieldToConceptIdMapping.js [hunk 6]
```diff
@@ -300,6 +300,30 @@ module.exports = {
...
+        tooBusy: 141450621,
+        incentiveTooLow: 576083042,
+        tooSick: 431428747,
+        unreliableInternetDevice: 121430614,
+        worriedAboutStudyResults: 523768810,
+        worriedStudyWillFindSomethingBad: 639172801,
+        privacyConcerns: 175732191,
+        donNotTrustGov: 150818546,  
+        donNotTrustRsrchrs: 624030581,  
+        donNotWantInfoSharedWithRsrchrs: 285488731,
```
Comment (we-ai) [inline utils/fieldToConceptIdMapping.js:312]: `doNot...`

**Comment by jhflorey** [comment]
@anthonypetersen we was missing some stub variable (all the reasons for refusal). So i have to add them.

image

**Review by Gbarra9 — APPROVED** [review]

**Review by anthonypetersen — APPROVED** [review]

**Pull request merged**